### DISCUSSION TITLE
3D videos for Apple Vision Pro

### DISCUSSION THREAD

**Discussion by douglascerny** [SublerApp/Subler#44] [Ideas]
Hello, I have been going through and converting all my 3D Blu-rays to digital files that are native MV-HEVC 3D videos for the Apple Vision Pro, not just SBS or TB split-view 3D videos. For reference, this is the software doing the conversion: https://github.com/cbusillo/BD_to_AVP . This creates a very specifically formatted .mov file with 360 metadata injected into it so that the Vision Pro knows it is 3D and how to handle it. However, I also want to have cover art and movie meta data added. I've tried using Subler to do this since i love the search capability for art and info, but because it re-transcribes the file, it breaks it. It not only isn't 3D anymore, but it doesn't even play. 

My ask is to have a way to either a) provide a mode to just append the metadata without any kind of conversion (akin to how the mac TV app does with these files), or b) allow the app to handle MV-HEVC 3D movies (much harder conversion details). 

Thanks!

**Comment by galad87** [comment]
Can you sure a small sample file so I can check what's going wrong?

**Comment by douglascerny** [comment]
Hey, I'm sorry but i'm not actually sure how to just send a small file. Any
tool I try to trim the 3D video in will do some sort of
transcribing/converting to save the trimmed video, so it won't get you what
you need. I can put a video in my Google Drive and send you a link, but is
there a way to send you that directly? Being that it would be a full movie,
I don't want to post a link publicly for piracy reasons. I'm a bit of a
github noob, so i apologize if knowing how to DM someone should be obvious
